### PR TITLE
Allows jp8 fuel to be found at military helipads

### PR DESCRIPTION
**Pull request by thelonestander**
#### Summary

Allowed jp8 to spawn at military helipads

SUMMARY: Balance "Allows jp8 fuel to be found at military helipads"

#### Purpose of change

I noticed while playing that jp8 fuel is nowhere to be found besides on crashed helicopters or in some helicopters by default. Using any other fuel for helicopters is not efficient, so I added fuel spawns to allow more usage of helicopters!  

#### Describe the solution

Just added a line of code to allow them to spawn!

#### Describe alternatives you've considered

Making Jp8 pumps, and adding them to airports, private runways, and military locations. Learning how to code better. 

#### Testing

Tested in game, loaded up vanilla world with no mods, spawned in a military helipad, and jp8 fuel spawned.

#### Additional context

image

### CODE REVIEW

**Comment by thelonestander** [comment]
Also Chaos volt helped me with all this stuff. Thanks man.

**Comment by Firestorm01X2** [comment]
It seems that airport fuel pumps lack of JP8 as well.

File: data/json/itemgroups/supplies.json [hunk 1]
```diff
@@ -519,7 +519,8 @@
     "entries": [
       { "item": "gasoline", "charges": [ 0, 50000 ], "container-item": "30gal_drum", "prob": 30 },
       { "item": "diesel", "charges": [ 0, 50000 ], "container-item": "30gal_drum", "prob": 10 },
-      { "item": "motor_oil", "charges": [ 0, 10000 ], "container-item": "jerrycan", "prob": 5 }
+      { "item": "motor_oil", "charges": [ 0, 10000 ], "container-item": "jerrycan", "prob": 5 },
+	  { "item": "jp8", "charges": [ 0, 50000 ], "container-item": "30gal_drum", "prob": 20 }
```
Comment (chaosvolt) [inline data/json/itemgroups/supplies.json:523]:
```suggestion
      { "item": "jp8", "charges": [ 0, 50000 ], "container-item": "30gal_drum", "prob": 20 }
```

**Comment by Coolthulhu** [comment]
Math:
* 12 barrel spawns
* 75% chance per spawn, so effectively 9 spawns
* 20/65=31% chance of spawn to be jp8, so ~2.8 spawns
* 2.8 * (50000 / 2) = ~70000 units of jp8 on average

>  Using any other fuel for helicopters is not efficient, so I added fuel spawns to allow more usage of helicopters!

Why?

**Comment by chaosvolt** [comment]
They might need the quantity spawned in barrels will be a windfall when it does spawn, otherwise alternatives might include adjusting the weights or other tweaks.

Side note, the fuel value of JP8 is actually slightly lower than diesel if I recall, so it's not actually more useful as fuel than diesel. I assume the idea is that it's supposed to be a "infrequent spawn but spawns in bulk when it does spawn, making it a nice windfall" sort of situation.

**Comment by AngelicosPhosphoros** [comment]
This helipads have gas pumps and diesel pumps. Can we add jp8 pumps on military objects as well?

**Comment by thelonestander** [comment]
As to the why, I figured since its a military location and has helicopters, it would make sense it would spawn some fuel barrels of Jp8. 

As for the math, I figured the military would probably have taken Almost all of their fuel with them, and any fuel left behind would be for the player. For the in-game reason, I didn't want there to be too much fuel spawned as to not rely solely on helicopters to get around.

**Comment by thelonestander** [comment]
> This helipads have gas pumps and diesel pumps. Can we add jp8 pumps on military objects as well?

I tried to add them, but I could only get them to spawn fuel next to the fuel pump. I'm afraid its past my level of skill to add.

**Comment by DrPariah** [comment]
Don't have to worry about people relying on copters too much. Turrets are still exceptionally dangerous in BN, and tussling with the military dead at the site itself after dealing with the turrets means risk/reward is pretty fair.

**Comment by Coolthulhu** [comment]
@Krwak protip: you can use triple backtick, followed by language (or nothing), for a multiline code block.

For example:
````
```JSON
{
  "key": "value"
}
```
````
Will become:
```JSON
{
  "key": "value"
}
```

Edited your comment to show how it looks.

**Comment by thelonestander** [comment]
> Math:
> 
> * 12 barrel spawns
> * 75% chance per spawn, so effectively 9 spawns
> * 20/65=31% chance of spawn to be jp8, so ~2.8 spawns
> * 2.8 * (50000 / 2) = ~70000 units of jp8 on average
> 
> > Using any other fuel for helicopters is not efficient, so I added fuel spawns to allow more usage of helicopters!
> 
> Why?

Would you like me to increase the amount of Jp8?

**Pull request merged**